apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingStatusService.kt

```diff
@@ -68,6 +71,11 @@ class BehandlingStatusServiceImpl constructor(
         val behandling = hentBehandling(behandlingId)
         inTransaction {
             lagreNyBehandlingStatus(behandling.tilFattetVedtak())
+            oppgaveService.opprettNyOppgaveMedSakOgReferanse(
```

Comment:
Tror vi må se på dette som en egen ting og ikke blande det inn i statusoppdateringene men heller gjøre et separat kall for dette.